extension/src/extension.ts

```diff
@@ -243,6 +243,7 @@ export function activate(context: vscode.ExtensionContext): void {
   context.subscriptions.concat(troubleshootingFunctions);
   context.subscriptions.concat(powerShellFunctions);
   //context.subscriptions.push(disposable);
+  NABfunctions.runTaskItems();
```

Comment:
What happens if an error happens inside this function?  Will it stop the activation of the extension?
We might need to catch all and just continue, or what do you say?

Reply:
I think all errors are caught but I agree better safe than sorry.